docs/source/getting-started.rst

```diff
@@ -94,7 +94,93 @@ or for ``Ninja``:
 
 .. note::
 
-        Loading the bitstream into an FPGA can be done outside of the Symbiflow with tools that support the target architecture, such as for example `Vivado` or `xc3sprog` for Xilinx 7-Series FPGAs or ``iceprog`` for Lattice iCE40.
+        Loading the bitstream into an FPGA can be done outside of the Symbiflow with tools that support the target architecture. A good example is `openFPGALoader` or others such as `Vivado` and `xc3sprog` for Xilinx 7-Series FPGAs or ``iceprog`` for Lattice iCE40.
```

Comment:
I recommend not to add references to other programming tools OR to do it "properly". With proper I mean something such as:

> There are multiple tools for loading bitstreams into FPGA development boards. Typically, each tool supports an specific target family or the lines of products of a vendor. Some of the most known are the following:
>
> - Xilinx/Digilent's iMPACT, ChipScope, EDK, Hardware Manager, xc3sprog, etc.
> - Lattice's Standalone Programmer.
> - OpenOCD.
> - dfu-util.
> - Icestorm's iceprog.
> - IceSugar's icesprog.
> - Greg Davill's ecpprog.
> - fujprog for ULX2/ULX3.
>
> Fortunately, since 2019, a universal utility named openFPGALoader is being developed, which aims to provide a replacement for most of those. It supports many different boards with FPGAs based on the architectures including xc7, ECP5, iCE40 and many more. It can utilize a variety of the programming adapters based on JTAG, DAP interface, ORBTrace, DFU and FTDI chips.

Moreover, that list might be contributed in https://github.com/hdl/constraints or as a section/tag in https://github.com/hdl/awesome, so that it could be just referenced from here.

Reply:
https://github.com/hdl/constraints/tree/main/prog